packages/orbit-components/src/Inline/__tests__/index.test.js

```diff
@@ -0,0 +1,89 @@
+// @flow
+import React from "react";
+import { screen, render } from "@testing-library/react";
+
+import theme from "../../defaultTheme";
+import Inline from "..";
+
+const dataTest = "test";
+
+const tokens = {
+  none: "",
+  XXXsmall: theme.orbit.spaceXXXSmall,
+  XXsmall: theme.orbit.spaceXXSmall,
+  Xsmall: theme.orbit.spaceXSmall,
+  small: theme.orbit.spaceSmall,
+  medium: theme.orbit.spaceMedium,
+  large: theme.orbit.spaceLarge,
+  Xlarge: theme.orbit.spaceXLarge,
+  XXlarge: theme.orbit.spaceXXLarge,
+  XXXlarge: theme.orbit.spaceXXXLarge,
+};
+
+const Elements = () => (
+  <>
+    <div data-test="child-0">kek</div>
+    <div>bur</div>
+    <div>cheburek</div>
+    <div>blin</div>
+  </>
+);
+
+describe("#Inline", () => {
+  it("should have props", () => {
+    render(
+      <Inline align="center" justify="start" dataTest={dataTest}>
+        <Elements />
+      </Inline>,
+    );
+    expect(screen.getByTestId(dataTest)).toBeInTheDocument();
+    const styles = screen.getByTestId(dataTest);
+    expect(styles).toHaveStyle({ alignItems: "center" });
+    expect(styles).toHaveStyle({ justifyContent: "flex-start" });
+  });
+
+  it("should have spacings", () => {
+    const spacings = Object.keys(tokens).map(token => (
+      <Inline key={token} align="start" justify="center" spacing={token} dataTest={dataTest}>
+        <div data-test={token} style={{ marginLeft: tokens[token] }}>
+          kek
+        </div>
+        <div>bur</div>
+      </Inline>
+    ));
```

Comment:
I testing this with [`it.each`](https://jestjs.io/docs/en/api#testeachtablename-fn-timeout) instead, for example:

```js
it.each(Object.entries(tokens))('should have expected spacing for token "%s"', (token, spacing) => {
  render(
    <Inline align="start" justify="center" spacing={token} dataTest={dataTest}>
      <div data-test={token} style={{ marginLeft: tokens[token] }}>
        kek
        </div>
      <div>bur</div>
    </Inline>
  );
  expect(screen.getByTestId(dataTest)).toHaveStyle({ marginLeft: spacing });
});
```

😎 (I didn't check whether this works.)

Reply:
works, thx :)